base_location_code_program_territory/tests/test_base_location_code_program_territory.py

```diff
@@ -0,0 +1,166 @@
+from odoo.tests.common import TransactionCase
+
+
+class TestPartnerCodeProgramSale(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
```

Comment:
I think you could have planned it to inherit one of the test classes from the previous PRs and not duplicate the setup code. But you can leave it as is.